### PR TITLE
rename 'portable' to 'pickable'

### LINKED ISSUE #1695: "Portable" property is a misnomer
Entities that are `portable` may not be `grab`ed or `harvest`ed, or obtained as the result of `drill`.  However, such entities can still exist in the inventory and certainly don't preclude `move`ment.  Non-portable entities may already be specified in the inventory at the outset of a scenario, or they may enter the inventory by via `make`, or be `salvage`ed, or be `give`n, etc.  In fact, certain scenario mechanics already depend on the behavior being so.

Perhaps a better term for the property would be `pickable`?

### PR DESCRIPTION
**Pull request by kostmo**
Closes #1695.

Updating occurrences was pretty easy:

    yq --inplace '(.[].properties[] | select(. == "portable")) |= "pickable"' data/entities.yaml

and

    find data/scenarios -type f -name '*.yaml' -print0 | xargs --max-args 1 --null yq --inplace '(select(has("entities")) | .entities[] | select(has("properties")) | .properties[] | select(. == "portable")) |= "pickable"'

### CODE REVIEW

**Comment by byorgey** [comment]
When we merge this we should also remember to regenerate affected wiki cheat sheets.

File: data/scenarios/Testing/1034-custom-attributes.yaml [hunk 1]
```diff
@@ -22,8 +22,7 @@ attrs:
     fg: "#ff0000"
     bg: ffff00
   - name: cyanOnMagenta
-    fg: "#00ffff
-    bg: "#ff00ff"
+    fg: "#00ffff bg: " #ff00ff"
```
Comment (byorgey) [inline data/scenarios/Testing/1034-custom-attributes.yaml:25]: It looks like there was a missing double quote.  I don't know if it worked before, but it certainly doesn't now! :smile:

File: data/scenarios/Testing/1218-stride-command.yaml [hunk 2]
```diff
@@ -8,48 +8,17 @@ objectives:
...
-    grab;
-    return ();
-  } {
-    log "Could not grab southern flower";
-  };
-  
-  turn east;
-  stride 5;
-  grab;
+solution: "stride 0;\nstride (-1);\n\nturn north;\ntry {\n  stride 5;\n  grab;\n  return ();\n} {\n  log \"Could not grab northern flower\";\n};\n\nturn west;\ntry {    \n  stride 1;\n} {\n  log \"Could not stride west\";\n};\n\nturn south;\ntry {\n  stride 5;\n  grab;\n  return ();\n} {\n  log \"Could not grab southern flower\";\n};\n\nturn east;\nstride 5;\ngrab;\n"
```
Comment (byorgey) [inline data/scenarios/Testing/1218-stride-command.yaml:21]: Eek, why did the solution get reformatted like this?  Can we fix it?

File: data/scenarios/Testing/1575-structure-recognizer/1575-ensure-disjoint.yaml [hunk 3]
```diff
@@ -15,27 +15,15 @@ objectives:
     goal:
       - |
         Build 2 of the same structure
-    condition: |
-      foundStructure <- structure "chessboard" 0;
-      return $ case foundStructure (\_. false) (\fs. 
-        let boardCount = fst fs in
-        boardCount >= 2;
-      );
+    condition: "foundStructure <- structure \"chessboard\" 0;\nreturn $ case foundStructure (\\_. false) (\\fs. \n  let boardCount = fst fs in\n  boardCount >= 2;\n);\n"
```
Comment (byorgey) [inline data/scenarios/Testing/1575-structure-recognizer/1575-ensure-disjoint.yaml:18]: Here's another case where a program got reformatted as an escaped string.

File: data/scenarios/Testing/1575-structure-recognizer/1575-ensure-disjoint.yaml [hunk 4]
```diff
@@ -15,27 +15,15 @@ objectives:
...
         while the bases still possesses `silver`{=entity}
-    condition: |
-      robotHasSilver <- as base {has "silver"};
-
-      foundStructure <- structure "chessboard" 0;
-      return $ case foundStructure (\_. false) (\fs. 
-        let boardCount = fst fs in
-        boardCount >= 2 && robotHasSilver;
-      );
+    condition: "robotHasSilver <- as base {has \"silver\"};\n\nfoundStructure <- structure \"chessboard\" 0;\nreturn $ case foundStructure (\\_. false) (\\fs. \n  let boardCount = fst fs in\n  boardCount >= 2 && robotHasSilver;\n);\n"
```
Comment (byorgey) [inline data/scenarios/Testing/1575-structure-recognizer/1575-ensure-disjoint.yaml:26]: And another.

File: data/scenarios/Testing/1575-structure-recognizer/1575-ensure-single-recognition.yaml [hunk 5]
```diff
@@ -12,27 +12,15 @@ objectives:
     goal:
       - |
         Build 2 of the same structure
-    condition: |
-      foundStructure <- structure "chessboard" 0;
-      return $ case foundStructure (\_. false) (\fs. 
-        let boardCount = fst fs in
-        boardCount >= 2;
-      );
+    condition: "foundStructure <- structure \"chessboard\" 0;\nreturn $ case foundStructure (\\_. false) (\\fs. \n  let boardCount = fst fs in\n  boardCount >= 2;\n);\n"
```
Comment (byorgey) [inline data/scenarios/Testing/1575-structure-recognizer/1575-ensure-single-recognition.yaml:15]: Another one.

File: data/scenarios/Testing/1575-structure-recognizer/1575-ensure-single-recognition.yaml [hunk 6]
```diff
@@ -12,27 +12,15 @@ objectives:
...
         while the bases still possesses `gold`{=entity}
-    condition: |
-      robotHasGold <- as base {has "gold"};
-
-      foundStructure <- structure "chessboard" 0;
-      return $ case foundStructure (\_. false) (\fs. 
-        let boardCount = fst fs in
-        boardCount >= 2 && robotHasGold;
-      );
+    condition: "robotHasGold <- as base {has \"gold\"};\n\nfoundStructure <- structure \"chessboard\" 0;\nreturn $ case foundStructure (\\_. false) (\\fs. \n  let boardCount = fst fs in\n  boardCount >= 2 && robotHasGold;\n);\n"
```
Comment (byorgey) [inline data/scenarios/Testing/1575-structure-recognizer/1575-ensure-single-recognition.yaml:23]: And another one.

File: src/Swarm/TUI/View.hs [hunk 7]
```diff
@@ -1195,7 +1195,7 @@ displayProperties = displayList . mapMaybe showProperty
   showProperty Unwalkable = Just "blocking"
   showProperty Opaque = Just "opaque"
   -- Most things are portable so we don't show that.
```
Comment (byorgey) [inline src/Swarm/TUI/View.hs:1197]: Should change the word `portable` in the comment too.

**Review by byorgey — APPROVED** [review]
I see that we are doing #845 now too. :smile:  I'm fine with that. Though I did notice a few things that need cleaning up.  I didn't look at absolutely every single change so we should try searching for additional instances of the things I pointed out.

**Comment by kostmo** [comment]
> I see that we are doing #845 now too.

Will finish #1711 before merging this.

File: data/scenarios/Testing/1034-custom-attributes.yaml (hunk 1 above)
Comment (kostmo) [reply data/scenarios/Testing/1034-custom-attributes.yaml:25]: Fix in #1712.

**Review by byorgey — APPROVED** [review]

**Pull request merged**